service/telemetry/config.go

```diff
@@ -103,3 +104,16 @@ type MetricsConfig struct {
 	// Address is the [address]:port that metrics exposition should be bound to.
 	Address string `mapstructure:"address"`
 }
+
+// TracesConfig defines the configurable settings for service telemetry traces.
+type TracesConfig struct {
+	// Configuration related to propagation of collector traces.
+	Propagation PropagationConfig `mapstructure:"propagation"`
+}
+
+// PropagationConfig defines the configuration related to propagation of collectors traces
+type PropagationConfig struct {
+	// Enabled sets context propagation of collector traces to downstream
+	// services consuming collectors export requests.
+	Enabled bool `mapstructure:"enabled"`
```

Comment:
See [my comment](https://github.com/open-telemetry/opentelemetry-collector/pull/5670#discussion_r918731880) elsewhere.  I don't think `enabled` is the right option here.  I'm not sure any option is required given that propagation can be disabled by using the `none` "propagator".